docs/data-sources/organization_project.md

```diff
@@ -0,0 +1,49 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "aiven_organization_project Data Source - terraform-provider-aiven"
+subcategory: ""
+description: |-
+  Gets information about an Aiven project.
+  This resource is in the beta stage and may change without notice. Set
+  the PROVIDER_AIVEN_ENABLE_BETA environment variable to use the resource.
+---
+
+# aiven_organization_project (Data Source)
+
+Gets information about an Aiven project. 
+
+**This resource is in the beta stage and may change without notice.** Set
+the `PROVIDER_AIVEN_ENABLE_BETA` environment variable to use the resource.
+
+## Example Usage
+
+```terraform
+data "aiven_organization_project" "example" {
+  project_id      = aiven_organization_project.foo.project_id
+  organization_id = aiven_organization_project.foo.organization_id
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `organization_id` (String) ID of an organization. Changing this property forces recreation of the resource.
+- `project_id` (String) Unique identifier for the project that also serves as the project name. Once set, cannot be changed.
+
+### Read-Only
+
+- `billing_group_id` (String) Billing group ID to assign to the project. It's required when moving projects between organizations.
+- `id` (String) The ID of this resource.
+- `parent_id` (String) Link a project to an [organization, organizational unit,](https://aiven.io/docs/platform/concepts/orgs-units-projects) or account by using its ID. To set up proper dependencies please refer to this variable as a reference.
+- `tag` (Set of Object) Tags are key-value pairs that allow you to categorize projects. (see [below for nested schema](#nestedatt--tag))
```

Comment:
In the API this is tags (plural). Should it be plural in TF as well?

Reply:
In the TF the tag will have a little bit different syntax to be consistent with other resources.
The schema will look like example below. So each tag is represented as a separate object.

```hcl
resource "aiven_organization_project" "foo" {
  organization_id = var.organization_id
  project_id = "example"
  billing_group_id = var.billing_group_id
    technical_emails = [
      "john.doe+1@gmail.com",
      "john.doe+2@gmail.com",
      "john.doe+3@gmail.com"
    ]

    tag {
      key = "value_1"
      value = "test_1"
    }

    tag {
      key = "value_2"
      value = "test_2"
    }
}
```